### PR TITLE
Feature/redesign

### PR DESCRIPTION
**Pull request by julianorinyol**

### CODE REVIEW

File: src/app/about/about.component.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="main">
```
Comment (julianorinyol) [inline src/app/about/about.component.html:1]: Use shared component for this
MainLayoutComponent

File: src/app/about/about.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AboutComponent } from './about.component';
+
+describe('MainLayoutComponent', () => {
```
Comment (julianorinyol) [inline src/app/about/about.component.spec.ts:5]: change to 
`describe('AboutComponent'

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -36,17 +84,14 @@ const routes: Routes = [
...
-    component: LoginPageComponent,
-  },
   {
     path: 'archive/:id',
     component: ArchiveComponent,
     canActivate: [AuthGuard],
   },
   { path: 'my-file', component: MyFileComponent, canActivate: [AuthGuard] },
   { path: 'issues', component: IssuesComponent },
+  { path: 'about', component: AboutComponent },
```
Comment (julianorinyol) [inline src/app/app-routing.module.ts:94]: add MainLayoutComponent here

File: src/app/app.component.ts [hunk 4]
```diff
@@ -26,14 +26,14 @@ export class AppComponent {
...
-    if(window.innerWidth <= 1000){
-      this.dialog.open(MobileDialogComponent, {
-        position: {
-          top: '50px',
-          left: '50px'
-        },
-        width: '300px',
-      });
-    }
+    // if(window.innerWidth <= 1000){
```
Comment (julianorinyol) [inline src/app/app.component.ts:29]: delete comments

File: src/app/app.module.ts [hunk 5]
```diff
@@ -25,14 +30,17 @@ import { environment } from 'src/environments/environment';
...
+import { AboutComponent } from './about/about.component';
 
 // eslint-disable-next-line prefer-arrow/prefer-arrow-functions
 export function createTranslateLoader(http: HttpClient) {
   return new TranslateHttpLoader(http, './assets/i18n/', '.json');
 }
 
 @NgModule({
-  declarations: [AppComponent, MainLayoutComponent],
+  declarations: [AppComponent, MainLayoutComponent, FooterOnlyLayoutComponent, AboutComponent],
```
Comment (julianorinyol) [inline src/app/app.module.ts:43]: are these Declarations needed? doouble check

File: src/app/authentication-page/authentication-page.component.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
+<div class="authentication-page container">
```
Comment (julianorinyol) [inline src/app/authentication-page/authentication-page.component.html:1]: get rid of or renamed `container` and `container-fluid`. as currently there is no general styles for those.

File: src/app/authentication-page/authentication-page.component.html [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+        </div>
+
+        <div class="full-width-container">
+            <div class="wrapper">
+                <div class="case-list-container">
+                    <h3>Die letzten gelosten Falle</h3>
+                    <app-case-list [cases]=cases></app-case-list>
+                    <button type="submit" class="">Weitere gelöste Fälle ansehen</button>
+                </div>
+                <div class="unsere_partner_section">
```
Comment (julianorinyol) [inline src/app/authentication-page/authentication-page.component.html:53]: rename class

File: src/app/authentication-page/authentication-page.component.scss [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+        text-align:center;
+        margin-bottom: 40px
+    }
+
+    .passwort_vergessen {
+        margin: 0 0px 0 110px;
+    }
+    .line {
+        height: 1px;
+        margin: 40px 70px 20px 70px;
```
Comment (julianorinyol) [inline src/app/authentication-page/authentication-page.component.scss:24]: get rid of weird margin

File: src/app/authentication-page/authentication-page.component.scss [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+        padding-bottom: 70px;
+    }
+
+    .logo {
+        text-align:center;
+        margin-bottom: 40px
+    }
+
+    .passwort_vergessen {
+        margin: 0 0px 0 110px;
```
Comment (julianorinyol) [inline src/app/authentication-page/authentication-page.component.scss:20]: change away from using margin or just set the margin that is needed

File: src/app/authentication-page/authentication-page.component.scss [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+    }
+    .line {
+        height: 1px;
+        margin: 40px 70px 20px 70px;
+        background-color: #fff;
+    }
+
+    .login_form_buttom_text {
+        text-align: center;
+        color: #ffffff;
```
Comment (julianorinyol) [inline src/app/authentication-page/authentication-page.component.scss:30]: name colors,  use sass variables

File: src/app/authentication-page/authentication-page.component.scss [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+
+    .unsere_partner_section .paragraph {
+        margin-top: 70px;
+    }
+
+    .orange {
+        color: orange;
+    }
+
+
```
Comment (julianorinyol) [inline src/app/authentication-page/authentication-page.component.scss:116]: remove all extra lines

File: src/app/confirm-email-page/components/confirm-email-form/confirm-email-form.component.scss [hunk 12]
```diff
@@ -0,0 +1,37 @@
+@import "../../../../assets/scss/_buttons.scss";
```
Comment (julianorinyol) [inline src/app/confirm-email-page/components/confirm-email-form/confirm-email-form.component.scss:1]: change all spacing to 2 spaces

File: src/app/core/dialogs/menu-dialog/menu.component.scss [hunk 13]
```diff
@@ -0,0 +1,283 @@
+@charset "utf-8";
+@import "../../../../assets/scss/_initializer.scss";
+
+// .mat-list-item-content {
```
Comment (julianorinyol) [inline src/app/core/dialogs/menu-dialog/menu.component.scss:4]: remove comment

File: src/app/forgot-password-page/components/forgot-password-form/forgot-password-form.component.scss [hunk 14]
```diff
@@ -0,0 +1,37 @@
+@import "../../../../assets/scss/_buttons.scss";
+.authentication-form-container {
+	display: flex;
```
Comment (julianorinyol) [inline src/app/forgot-password-page/components/forgot-password-form/forgot-password-form.component.scss:3]: spacing 2 spaces

File: src/app/forgot-password-page/components/forgot-password-form/forgot-password-form.component.ts [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+@Component({
+  selector: 'app-forgot-password-form',
+  templateUrl: './forgot-password-form.component.html',
+  styleUrls: ['./forgot-password-form.component.scss']
+})
+
+export class ForgotPasswordFormComponent implements OnInit {
+  forgotPasswordForm: FormGroup;
+  invalid: boolean;
+  // closeResult = {
```
Comment (julianorinyol) [inline src/app/forgot-password-page/components/forgot-password-form/forgot-password-form.component.ts:28]: * remove comments
* forgotPasswordForm 
should have 'public'

File: src/app/login-page/components/login-form/login-form.component.scss [hunk 16]
```diff
@@ -1,29 +1,37 @@
 @import "../../../../assets/scss/_buttons.scss";
-.login-form-card {
-	border-radius: 30px;
+.authentication-form-container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
 
-	.login-form-card--contents {
+	.authentication-form {
```
Comment (julianorinyol) [inline src/app/login-page/components/login-form/login-form.component.scss:7]: spacing 2 spaces

**Comment by PotSchmoell** [comment]
Thanks for your work! Great to see the new design coming to life!

I have a few remarks after an initial testing:

Header:
- [ ] click on the header logo should redirect to start page (`/`)
- [ ] The header should be present on all pages except for authentication pages (--> add header on review, archive, etc.)
- [ ] The icon for "Detektiv werden" is missing/wrong (showing a TM sign) 
- [ ] Menu mobile: Can you try centering the list? I think this should look better?

Footer:
- [ ] Can you try to center the content in the single column layout (mobile). I think this should look better.

Auth-Pages:
- [ ] Clicking on the logo should redirect to start page
- [ ] The buttons on the bottom (e.g. "Detektiv werden" on login page) have no "cursor pointer"
- [ ] Pressing enter should submit the form 
- [ ] Mobile: Please increase the distance between the border of the white "form box" and the input lines for mail and pwd. It should look more like on the desktop version

**Comment by PotSchmoell** [comment]
Maybe you can look into putting related components (e.g. all authentication related stuff) in one folder/module.

**Comment by julianorinyol** [comment]
On registeration:
* try to minimize the times you login.  
register-> login -> verify   done! 
instead of register-> login -> verify   -> login -> done!

**Comment by julianorinyol** [comment]
Make sure case-list  is not wider than the screen on mobile.

**Pull request closed without merging**